tests/system/src/test/java/test/javafx/scene/InitializeJavaFXTest.java

```diff
@@ -0,0 +1,84 @@
+package test.javafx.scene;
```

Comment:
1. Missing Copyright header block
2. Other platform startup tests are in `test.com.sun.javafx.application`; can you move this test there as well?

Reply:
done